dattri/algorithm/influence_function.py

```diff
@@ -97,6 +97,15 @@ def transformation_on_query(
 class IFAttributorArnoldi(BaseInnerProductAttributor):
     """The inner product attributor with Arnoldi inverse hessian transformation."""
 
+    def __init__(self, *args, **kwargs) -> None:
+        """Initialize the attributor.
+
+        # noqa: DAR101
+        """
+        super().__init__(*args, **kwargs)
+        # init arnoldi projector
+        self.arnoldi_projector = None
```

Comment:
actually, I am wondering whether it is possible to make this more clean. @TheaperDeng do you have any thought?

Reply:
I guess this is just for the code style checking right?

Reply:
yeah, but I mean whether it is possible to append one more variable (the projector here) without writing all these stuffs

Reply:
I think for users who are using `dattri`, they may directly forget all these thing since they won't need the code style checking.

For us, I guess some options are
1. just ignore this code style checking rule
2. Use property https://www.programiz.com/python-programming/property

Reply:
Use other workaround to avoid this `init` overriding